scipy/io/wavfile.py

```diff
@@ -88,8 +95,11 @@ def _read_data_chunk(fid, comp, noc, bits, is_big_endian, mmap=False):
         fmt = '>I'
     else:
         fmt = '<I'
+
+    # Size of the data subchunk in bytes
     size = struct.unpack(fmt, fid.read(4))[0]
 
+    # Number of bytes per sample
     bytes = bits//8
```

Comment:
It might be worth renaming this variable to something like `bytes_per_sample`, while we're doing maintenance, to avoid shadowing the builtin name.